search/includes/classes/queue/class-cron.php

```diff
@@ -147,10 +148,22 @@ public function filter_cron_schedules( $schedule ) {
 	 *
 	 * This is the cron hook for indexing a batch of objects
 	 *
-	 * @param array $options Containing max_id and min_id keys
+	 * @param array $options Containing either max_id and min_id or option keys
 	 */
 	public function process_jobs( $options ) {
-		$jobs = $this->queue->get_jobs_by_range( $options['min_id'], $options['max_id'] );
+		if ( ! empty( $options['option'] ) ) {
+			$job_ids = get_option( $options['option'] );
```

Comment:
Using a separate option to store Ids to avoid it being in alloptions is smart I think :+1: